<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>787</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>超级板标签码自动生成</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>app_icon_32.ico</normaloff>app_icon_32.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>210</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>初始化</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_18">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>选择sheet：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_sheets">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_select">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <italic>false</italic>
                   </font>
                  </property>
                  <property name="text">
                   <string>确定</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>按钮</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QPushButton" name="pushButton_execl_head">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="font">
               <font>
                <italic>false</italic>
                <kerning>false</kerning>
               </font>
              </property>
              <property name="text">
               <string>设置execl头</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>操作日志</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QTextEdit" name="textEdit_optlog">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="text">
            <string>操作状态:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_status_result">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_status">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="uploadProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>录入信息栏</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <family>Aharoni</family>
               <pointsize>16</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string>起始ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_startid">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>false</italic>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="text">
            <string>需求个数</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLineEdit" name="lineEdit_need_num">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <italic>false</italic>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>产品型号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_type">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>产品大小</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_size">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>产品颜色</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_color">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>电池类型</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_bat">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>防水等级</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_water">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>输入电压</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_v">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>输入电流</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_i">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>电池容量</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_cap">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <family>Aharoni</family>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <italic>false</italic>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>日期时间</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateEdit">
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_reset">
              <property name="minimumSize">
               <size>
                <width>185</width>
                <height>380</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <italic>false</italic>
               </font>
              </property>
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_write">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>185</width>
                <height>380</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <italic>false</italic>
               </font>
              </property>
              <property name="text">
               <string>录入</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_close">
              <property name="minimumSize">
               <size>
                <width>185</width>
                <height>380</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <italic>false</italic>
               </font>
              </property>
              <property name="text">
               <string>关闭Execl</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_create"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_create">
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
